src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,19 @@
+.wb-eqht-css{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+}
+.wb-eqht-css .wb-eqht-grd{
+	display:flex;
+	flex-direction: column;
+	flex-grow:1;
+}
+.wb-eqht-css .hght-inhrt-css{
+	display: flex;
+	flex: 1 1 auto;
+	flex-direction: column;
+}
+.wb-eqht-css .hght-inhrt-css .eqht-trgt-css{
+	flex: 1 1 auto;
+}
```

Comment:
a CSS class is already a css selector. the word "css" is kind of overused here. The word "css" is also very technical. We will need a name more functional.

@GormFrank @luc-bertrand-hrsdc  do you have some suggestion? 

Me, my suggestion is to use `wb-eqht-grid` instead of `wb-eqht-css` and to rename `wb-eqht-grd` for the columns selector `[col-*]`

Reply:
`[class*="col-"]`

Reply:
This has been addressed